### PR TITLE
Update maven release plugin

### PR DESCRIPTION
**Pull request by fbelzunc**
@reviewbybees  CC @oleg-nenashev

### CODE REVIEW

**Comment by rsandell** [comment]
:bee:

**Comment by alecharp** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -84,6 +84,14 @@
           <failOnError>false</failOnError>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-release-plugin</artifactId>
+        <version>2.5.3</version>
+        <configuration>
+          <goals>deploy</goals>
```
Comment (jtnord) [inline pom.xml]: Why?

Comment (fbelzunc) [inline pom.xml]: right, I am correcting it in #25

**Pull request merged**